datetimerange/__init__.py

```diff
@@ -114,39 +119,47 @@ def __contains__(self, x):
         :type x: |datetime|/``DateTimeRange``/|str|
         :return: |True| if the ``x`` is within the time range
         :rtype: bool
-
         :Sample Code:
             .. code:: python
-
                 from datetimerange import DateTimeRange
-
                 time_range = DateTimeRange("2015-03-22T10:00:00+0900", "2015-03-22T10:10:00+0900")
                 print("2015-03-22T10:05:00+0900" in time_range)
                 print("2015-03-22T10:15:00+0900" in time_range)
-
                 time_range_smaller = DateTimeRange("2015-03-22T10:03:00+0900", "2015-03-22T10:07:00+0900")
                 print(time_range_smaller in time_range)
         :Output:
             .. parsed-literal::
-
                 True
                 False
                 True
-
         .. seealso::
             :py:meth:`.validate_time_inversion`
         """
 
         self.validate_time_inversion()
 
         if isinstance(x, DateTimeRange):
+            if (
```

Comment:
For me clearer would be something with [operator](https://docs.python.org/3.7/library/operator.html)

```
import operator

...
def __contains__(self, x):
    ...

        start_op = operator.le if self.start_inclusive else operator.lt
        end_op = operator.le if self.end_inclusive else operator.lt
        return start_op(x.start_datetime, self.start_datetime) and end_op(x.end_datetime, self.end_datetime)
```

also you have to check if `not self.start_inclusive and x.start_inclusive` because it's only one case out of 4:

given `c < d`
```
(a, c) in (a, d)
(a, c) in [a, d)
[a, c) not in (a, d)
[a, c) in [a, d)
```

Reply:
Ah cool I didn't know you can do that in python